content/docs/command-reference/import-url.md

```diff
@@ -134,11 +134,16 @@ produces a regular stage in `dvc.yaml`.
   default file name: `<file>.dvc`, where `<file>` is the desired file name of
   the imported data (`out`).
 
-- `--no-exec` - create the import `.dvc` file but don't download `url` (assumes
-  that the data source is valid). This is useful if you need to define the
-  project imports quickly, and download everything later (use `dvc update` to
-  finish the operation(s)); or if the target data already exist locally and you
-  want to "DVCfy" this state of the project (see also `dvc commit`).
+- `--no-exec` - create the import `.dvc` file but don't download `url` or get
+  checksums (assumes that the data source is valid). This is useful if you need
```

Comment:
checksums - no context for this term/idea. Not mentioned anywhere in the Description. Not even "hashes" are mentioned.

Reply:
Idea: could at least l'ink to https://dvc.org/doc/user-guide/project-structure/dvc-files#output-entries. And I'd prefer the term "file hash" but up to you.